client/tests/integration/triggers/time_trigger.rs

```diff
@@ -1,6 +1,6 @@
 #![allow(clippy::restriction)]
 
-use std::time::Duration;
+use std::{str::FromStr, time::Duration};
```

Comment:
```suggestion
use std::{str::FromStr as _, time::Duration};
```

If you're importing a trait just to be able to access it's methods, I prefer this format.